gammapy/background/cube.py

```diff
@@ -809,6 +809,38 @@ def plot_spectrum(self, coord, ax=None, style_kwargs=None):
             plt.close(fig)
         return ax
 
+    @property
+    def integral(self):
+        """Integral of the cube (`~astropy.units.Quantity`)"""
```

Comment:
OK, then just say that in the docstring, e.g. "the integral returned has the unit of the values in the cube times solid angle times energy". You could even mention the one or two common examples that we have.